framework/src/org/apache/cordova/engine/SystemWebViewClient.java

```diff
@@ -342,15 +341,14 @@ public AuthenticationToken removeAuthenticationToken(String host, String realm)
     /**
      * Gets the authentication token.
      *
-     * In order it tries:
+     * <p>In order it tries:
      * 1- host + realm
      * 2- host
      * 3- realm
-     * 4- no host, no realm
+     * 4- no host, no realm</p>
```

Comment:
To be formatted properly, this might need `<ol>` and `<li>` instead of just `<p>` (which will ignore newlines)

Reply:
Applied `<ol>` and `<li>` tags.